config/autoload/global.php

```diff
@@ -0,0 +1,17 @@
+<?php
+return [
+    'zf-content-negotiation' => [
+        'selectors' => [
+            'Foo' => [
+                'Zend\\View\\Model\\JsonModel' => [
```

Comment:
I'll test; I know that the admin writes to this file, so I'm not sure how well that will work.